docker/Dockerfile

```diff
@@ -65,16 +222,34 @@ RUN set -eux; \
     (if [ -d cache/dependency-check ]; then cp -r cache/dependency-check/ dependency-check/data/; fi); \
     dependency-check --updateonly
 
+# Node.js tools are not installed globally since it's either recommended to install locally,
+# or will only work with peer dependencies which are installed locally.
+WORKDIR /dependencies/js
+COPY lib/js/ ./
+RUN npm install && chmod +x jslinters
+    # Required for newer versions of eslint with support for TS linting via plugins
+    # typescript-eslint \
+    # @typescript-eslint/eslint-plugin
+ENV PATH /dependencies/js/node_modules/.bin:/dependencies/js:$PATH
+ENV JSTOOLS_HOME /dependencies/js
+
 # backend services
 COPY out/backend-*/*-jar-with-dependencies.jar /dependencies/
 
 WORKDIR /src
-
-RUN python -m pip install --no-cache-dir poetry==1.7.1
 COPY main.py readme.md pyproject.toml poetry.lock ./
 COPY agent/ agent/
 
 # install agent and python dependencies
 RUN poetry install
 
 ENTRYPOINT ["poetry", "run", "cli"]
+
+# install cloc command 
+RUN set -eux; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        cloc
+    
+# ---------- techdebt ----------
+RUN python -m pip install --no-cache-dir lizard
```

Comment:
For cloc, please check the exact version used in the backend (in dockerbuild-smp) and install that specific version using npm instead of apt. The system apt repos will only have a few versions available which may or may not match what we have in the backend.

Same goes for lizard, please check the version currently installed and install that specific version.